boa/src/syntax/lexer/template.rs

```diff
@@ -23,7 +23,7 @@ macro_rules! template_match {
 pub(super) struct TemplateLiteral;
 
 impl TemplateLiteral {
-    /// Creates a new string literal lexer.
+    /// Creates a new template literal lexer.
     pub(super) fn new() -> Self {
         Self {}
     }
```

Comment:
I don't think we need a `new()` function for a structure with no members, right?

Reply:
Isn't it best to always provide a new function to hide the internal structure of a struct?

Reply:
Hmmm if it's a unit structure, I don't think it's needed.

Reply:
Fixed